tests/test_image_transforms.py

```diff
@@ -307,6 +308,110 @@ def test_center_crop(self):
         self.assertEqual(cropped_image.shape, (300, 260, 3))
         self.assertTrue(np.allclose(cropped_image, expected_image))
 
+    def test_crop(self):
+        image = np.random.randint(0, 256, (3, 224, 224))
+
+        # Test that exception is raised if crop out of bounds and padding is not specified
+        with self.assertRaises(ValueError):
+            crop(image, 0, 0, 300, 300, data_format="channels_last")
+
+        # Crop box within the image bounds
+        # *-----------*
+        # |   +----+  |
+        # |   ||||||  |
+        # |   +----+  |
+        # |           |
+        # *-----------*
```

Comment:
I love these pictures. But maybe mentioning what the 0 and 5 are. And which one is the image and another is the box.